### PR TITLE
git-fame: init at 2.5.2

### PR DESCRIPTION
**Pull request by expipiplus1**
If some ruby boffin could help with this that would be super. At the moment the
git-fame binary doesn't end up in `/bin`. calling
`/nix/store/jxq90d7jlfj1d457lkasmi03g9l0vykh-ruby2.3.3-git_fame-2017-01-14/lib/ruby/gems/2.3.0/bundler/gems/git-fame-rb-64748dfb18b0/bin/git-fame`
works however.

### CODE REVIEW

**Comment by joachifm** [comment]
cc @zimbatm @Mic92 re ruby packaging

File: pkgs/applications/version-management/git-and-tools/git-fame/gemset.nix [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+    };
+    version = "1.1.5";
+  };
+  git_fame = {
+    source = {
+      fetchSubmodules = false;
+      rev = "64748dfb18b0a892e8b01483a5ebc5282c594a5d";
+      sha256 = "1vmhnmwp5l5d1k7syfn6p2nlqa59schkzzlk7xk2khzc5xzq18zf";
+      type = "git";
+      url = "git://github.com/oleander/git-fame-rb.git";
```
Comment (zimbatm) [inline pkgs/applications/version-management/git-and-tools/git-fame/gemset.nix]: What is the reason to use a git version, is release v2.5.2 too old?

File: pkgs/applications/version-management/git-and-tools/git-fame/default.nix [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{ lib, bundlerEnv, ruby, fetchFromGitHub }:
+
+bundlerEnv rec {
+  name = "git-fame-${version}";
+
+  version = "2017-01-14";
+  inherit ruby;
+  gemdir = ./.;
+
+  pname = "git_fame";
```
Comment (zimbatm) [inline pkgs/applications/version-management/git-and-tools/git-fame/default.nix]: it's possible that setting both the name and pname doesn't interact very well. If the ruby gem is called "git_fame" then it's better to name the package the same way.

File: pkgs/applications/version-management/git-and-tools/git-fame/Gemfile [hunk 3]
```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
+
+# Specify your gem's dependencies in git_fame.gemspec
+gemspec
```
Comment (zimbatm) [inline pkgs/applications/version-management/git-and-tools/git-fame/Gemfile]: Something's missing here. It should be `gem "git_fame"`

**Comment by expipiplus1** [comment]
@zimbatm I've applied your changes, however running `$(nix-build --no-out-link -A gitAndTools.git-fame)/bin/git-fame` doesn't work; running `$(nix-build --no-out-link -A gitAndTools.git-fame)/lib/ruby/gems/2.3.0/gems/git_fame-2.5.2/bin/git-fame` does though

```
$ $(nix-build --no-out-link -A gitAndTools.git-fame)/bin/git-fame
/nix/store/3d7a11zliyy2djcphlq5mx6rr1yxlll3-bundler-1.14.6/lib/ruby/gems/2.3.0/gems/bundler-1.14.6/lib/bundler/rubygems_integration.rb:408:in `block in replace_bin_path': can't find executable git-fame (Gem::Exception)
        from /nix/store/3d7a11zliyy2djcphlq5mx6rr1yxlll3-bundler-1.14.6/lib/ruby/gems/2.3.0/gems/bundler-1.14.6/lib/bundler/rubygems_integration.rb:434:in `block in replace_bin_path'
        from /nix/store/86ahl0dm6j72f34rh191dafp8jhda9kp-git-fame-2.5.2/bin/git-fame:18:in `<main>'
```

**Comment by zimbatm** [comment]
@expipiplus1 yeah there's something wrong with the ruby env. I've pushed a little change but it doesn't fit the main issue.

**Comment by Mic92** [comment]
fixed in 531a5b1afeb10ca4c347f2c7d8386e1187b456bf
Gemfile was not correctly updated.

**Pull request closed without merging**